src/plugins/intel_gna/src/transformations/insert_identity_layer.cpp

```diff
@@ -20,29 +21,33 @@ using namespace ov::intel_gna;
 using namespace ov::intel_gna::pass;
 using namespace ov::intel_gna::rt_info;
 using namespace ov::intel_gna::graph_utils;
+using namespace ov::pass::pattern;
+using namespace ov::op::util;
+
+using namespace std;
```

Comment:
I would remove this using to be consistent with rest sources, but it is not mandatory.
I think we don't use using for std namespace in any place. Are you sure you want to do this?
I don't understand why using namespace std would be beneficial, and at the same time not  adding using namespace ov( when all other nemespaces ov like namespaces are added)

Reply:
The obvious benefit to me is shorten code. I was trying to add "using namespace ov", but there were some ambiguity problems and I didn't have time to investigate it. So left as is for now.
Is "using namespace std" inconsistent with our coding rules?

Reply:
It is not inconsistent, but introducing this blurred a bit the PR, because dozes of lines were changes only because of that.

Problem I see that in most part of the code we don't use this using so it would be inconsistent with rest of the code.

Reply:
you can resolve this comment if you want stay with `using namespace std;`

Reply:
Yes, the code will be shorten but imho the most important for development to leave the code uniform, plus using std:: helps to identify stl functions in the code. But it doesn't block the PR

Reply:
Ok, reverted, however I don't see any advantage of keeping this in code as shared_ptr, vector, map, make_shared and so on are pretty clear.